src/utils/getArticleContents.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+const fluid = require("infusion");
+const jsdom = require("jsdom");
+const { JSDOM } = jsdom;
```

Comment:
could this be written as `const { JSDOM } = require("jsdom");`

Reply:
I agree! I'd just followed the syntax and instructions [presented in the jsdom instructions](https://github.com/jsdom/jsdom/blob/4c7eed155e421c3b261667b6312d4c89d2a74c1b/README.md?plain=1#L12), but this one-liner is much tidier. I've made the same change in `getResourcesLinks.js`